### PR TITLE
Consistency fixes

### PR DESCRIPTION
**Pull request by polyzen**

### CODE REVIEW

**Comment by polyzen** [comment]
Added more details to the commit messages. Not brave enough to also alphabetize all the lists.

**Comment by luciusmagn** [comment]
Oh my, I totally forgot about you, sorry 😅. I will check out your work when I get home today

**Comment by polyzen** [comment]
If the first commit is too dense to review, I could split it up.

**Comment by luciusmagn** [comment]
Howdy! The commit isn't too dense, it's just that I was hesitant due to proposed changes from @memoryruins. However, I came to the conclusion that your changes shouldn't really interfere with the proposals and I'd be happy to merge. Sorry for the delay

File: README.md [hunk 1]
```diff
@@ -238,6 +238,7 @@ See also [A comparison of operating systems written in Rust](https://github.com/
 
 * [OUISRC/muro](https://github.com/OUISRC/muro) — A new Forum use rust, rocket, diesel, postgresql, and responsive design [<img src="https://api.travis-ci.org/OUISRC/muro.svg?branch=master">](https://travis-ci.org/OUISRC/muro)
 
+
```
Comment (polyzen) [inline README.md]: This adds (back?) an extra newline. Should I rebase and force push the branch?

Comment (luciusmagn) [reply README.md]: I don't think it matters too much, but you are welcome to do so

Comment (polyzen) [reply README.md]: Done. :+1:

**Comment by luciusmagn** [comment]
Awesome, thanks!

**Pull request merged**